src/components/Icons/CollectionIcon.js

```diff
@@ -64,7 +77,7 @@ export default connect(mapStateToProps, mapDispatchToProps)(CollectionIcon);
 
 const styles = StyleSheet.create({
   text: {
-    color: 'white',
+    color: 'black',
```

Comment:
I agree that this is more readable, but `UI improvements sprint 1 and 2.pptx` shows white-on-gold.

...if the idea was "consistency" between icons, another alternative could be having some darker gold color, again.